test/test_site.rb

```diff
@@ -66,7 +66,7 @@ def session_id
 
   # Monkey patch API behavior to give static responses.
   class Nexpose::APIRequest
-    def self.execute(url, req, api_version='1.1')
+    def self.execute(url, _trust_store, _req, _api_version = '1.1')
```

Comment:
Should probably put this in the correct order for consistency, even though it's just test code.